src/ProtocolHandler/HttpHeaders/Header.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+
+namespace Chorizo.ProtocolHandler.HttpHeaders
```

Comment:
Probably doesn't need to be in the ProtocolHandler namespace.

Reply:
Added to Exchange namespace based on this [StackOverflow](https://stackoverflow.com/a/325358) post.